extensions/amp-bind/0.1/validator-amp-bind.protoascii

```diff
@@ -59,6 +59,8 @@ tags: {  # <amp-state> with json child
     name: "id"
     mandatory: true
   }
+  # <amp-bind>
```

Comment:
This PR makes me wonder if we even need two different flavors of amp-state tag. We could just make both the `src` attribute and the child JSON optional. If only one is specified then that's the only source for the state. If both are specified then the XHR state is deep merged with the JSON.